src/FastMoneyTurn.js

```diff
@@ -0,0 +1,33 @@
+import domUpdates from './domUpdates';
+import Turn from './Turn';
+
+class FastMoneyTurn extends Turn {
+  constructor() {
+    super();
+  }
+
+  endTurn(game) {
+    game.currentRound.currentPlayer === game.playerOne ? game.currentRound.currentPlayer = game.playerTwo : game.currentRound.currentPlayer = game.playerOne;
```

Comment:
I’d probably change this ternary into an if/else statement, it’s a little long and difficult to read at the moment. Ternaries should be kept very very simple, they should really only get as complex as something like this:

```js
let foo = bar ? true : false
```